falkordb/falkordb.py

```
@@ -121,9 +122,59 @@
+        # FalkorDB Cluster Params
+        self.cluster_error_retry_attempts=cluster_error_retry_attempts
+        self.startup_nodes=startup_nodes
+        self.require_full_coverage=require_full_coverage
+        self.reinitialize_steps=reinitialize_steps
+        self.read_from_replicas=read_from_replicas
+        self.dynamic_startup_nodes=dynamic_startup_nodes
+        self.url=url
+        self.address_remap=address_remap
+        self.sentinel_flag = False
+        self.cluster_flag = False
         if Is_Sentinel(conn):
```

Ruff

`Is_Sentinel` may be undefined, or defined from star imports

(F405)